packages/services/f-analytics/src/services/analytics.mapper.js

```diff
@@ -171,14 +171,18 @@ export const mapPageData = ({
     httpStatusCode,
     req
 } = {}) => {
-    pageData.group = featureName;
-    if (pageName) pageData.name = pageName;
     const conversationId = getCookie('x-je-conversation', req);
-    if (conversationId) pageData.conversationId = conversationId;
-    if (requestId) pageData.requestId = requestId;
-    if (httpStatusCode) pageData.httpStatusCode = httpStatusCode;
     const displaySize = getDisplaySize();
-    if (displaySize) pageData.display = displaySize;
     const orientation = getOrientation();
-    if (orientation) pageData.orientation = orientation;
+
+    return {
```

Comment:
\*chefs kiss\*

Reply:
@xander-marjoram I'm not a fan of overloading the `return`, now when stepping thru as a dev you have to exit the method before you can inspect the final value of `pageData`.

Reply:
Plus now we have one method passing by value and the others passing by ref which is not ideal